bash_it.sh

```diff
@@ -51,32 +61,42 @@ done
 
 # Load theme, if a theme was set
 if [[ ! -z "${BASH_IT_THEME}" ]]; then
+  _log_debug "Loading theme..."
   # Load colors and helpers first so they can be used in base theme
+  BASH_IT_LOG_PREFIX="themes: colors: "
   # shellcheck source=./themes/colors.theme.bash
   source "${BASH_IT}/themes/colors.theme.bash"
+  BASH_IT_LOG_PREFIX="themes: githelpers: "
   # shellcheck source=./themes/githelpers.theme.bash
   source "${BASH_IT}/themes/githelpers.theme.bash"
+  BASH_IT_LOG_PREFIX="themes: p4helpers: "
   # shellcheck source=./themes/p4helpers.theme.bash
   source "${BASH_IT}/themes/p4helpers.theme.bash"
+  BASH_IT_LOG_PREFIX="themes: base: "
   # shellcheck source=./themes/base.theme.bash
   source "${BASH_IT}/themes/base.theme.bash"
 
+  BASH_IT_LOG_PREFIX="lib: appearance: "
   # appearance (themes) now, after all dependencies
   # shellcheck source=./lib/appearance.bash
   source "$APPEARANCE_LIB"
 fi
 
-# Load custom aliases, completion, plugins
+BASH_IT_LOG_PREFIX="core: main: "
+_log_debug "Loading custom aliases, completion, plugins..."
 for file_type in "aliases" "completion" "plugins"
 do
   if [ -e "${BASH_IT}/${file_type}/custom.${file_type}.bash" ]
   then
+    BASH_IT_LOG_PREFIX="${file_type}: custom: "
     # shellcheck disable=SC1090
     source "${BASH_IT}/${file_type}/custom.${file_type}.bash"
   fi
 done
 
 # Custom
+BASH_IT_LOG_PREFIX="core: main: "
+_log_debug "Loading general custom files..."
 CUSTOM="${BASH_IT_CUSTOM:=${BASH_IT}/custom}/*.bash ${BASH_IT_CUSTOM:=${BASH_IT}/custom}/**/*.bash"
 for _bash_it_config_file in $CUSTOM
 do
```

Comment:
Set a prefix here as well? Plus debug message to show the loaded file's name?

Reply:
done